apps/02RockPaperScissors.js

```diff
@@ -5,16 +5,42 @@ var prompt = require('prompt');
 prompt.start();
 
 
+
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+    hand1 = hand1.toLowerCase();
+    hand2 = hand2.toLowerCase();
+    hand1 = hand1.trim();
+    hand2 = hand2.trim();
+
+    var handWin1 = 'Hand one wins!';
+    var handWin2 = 'Hand two wins!';
+    var handTie = "It's a tie!";
 
+    if (hand1 === hand2) {
+        return handTie;
+    } else if (hand1 === 'rock' && hand2 === 'scissors') {
+        return handWin1;
+    } else if (hand1 === 'rock' && hand2 === 'paper') {
+        return handWin2;
+    } else if (hand1 === 'scissors' && hand2 === 'paper') {
+        return handWin1;
+    } else if (hand1 === 'scissors' && hand2 === 'rock') {
+        return handWin2;
+    } else if (hand1 === 'paper' && hand2 === 'rock') {
+        return handWin1;
+    } else if (hand1 === 'paper' && hand2 === 'scissors') {
+        return handWin2;
+    } else {
+        return ('Invalid Input! Please enter rock, paper, or scissors to continue');
+    }
 }
 
```

Comment:
Very good if/else tree! If you want this code to be EVEN MORE fun, explore the 'or' operato ( || )  in the logic as well.

Example Pseudo code: 
`else if ( (hand1 === 'rock'' && hand2 === 'scissors') || (hand1==='paper' && hand2==='rock')) //etc
           return handWin1;`